Lesson 5 - CSS/code/exercises/exercise 1/theme.css

```diff
@@ -1,6 +1,66 @@
 .row {
     display: block;
+    font-family: 'Times New Roman', Times, serif;
+    margin: auto;
+    width: 1000px;
 }
-
-
-
+.column
+{
+    float: left;
+    width: 500px;
+}
+h1[title^="t1"]
+{
+    background-color: black; 
+    border-radius: 10px 10px 0 0;
+    color: white;
+    padding-top: 25px;
+    padding-bottom: 25px;
+    padding-left: 10px;
+}
+p[nazwa^="pierwszy"]:first-letter
+{
+    font-size: 100px;
+}
+p[nazwa^="pierwszy"]
+{
+    background-color: darkgrey;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-bottom: 20px;
+    text-align: justify;
+    border-radius: 0 0 10px 10px;
+    white-space: wrap;
+}
+h1[title^="t2"]
+{
+    background-color: darkorange; 
+    border-top: solid 5px black;

```

Comment:
* Margin is missing 
* Cat image border box is incorrect. 
* Missing ellipsis sign **(...)**

![image](https://cloud.githubusercontent.com/assets/4668423/14258799/5d04e5a0-faa4-11e5-8f92-6abdef887f88.png)